src/main/resources/View/AppView.fxml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.AnchorPane?>
+<?import javafx.scene.text.Font?>
+
+<AnchorPane maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="480.0"
+            prefWidth="640.0" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1"
+            fx:controller="pl.softwareforge.controller.AppViewController">
+    <children>
+        <Label fx:id="labelHello" alignment="CENTER" layoutX="195.0" layoutY="115.0" prefHeight="250.0"
```

Comment:
You might consider adding a label at the end, to make it easier to read, or leave it as it is, to make it easier to find all labels at once.